docs/development/design/3-DN-Retrieving_unique_graphs_from_Otel_data/class_diagrams.puml

```diff
@@ -57,7 +57,9 @@ IngestData ..> DataHolder
 OTELDataSource ..> OTelEvent
 JSONDataSource --> OTELDataSource
 SQLDataHolder --> DataHolder
-NodeModel --> SQLDataHolder
+NodeModel <.. SQLDataHolder
```

Comment:
Can we have this flipped so everything is pointing downwards rather than having an arrow going up. Just think it will make it look nicer